.github/workflows/container-tests.yml

```diff
@@ -3,7 +3,52 @@ on:
     types:
       - created
 jobs:
+  distgen-check:
+    name: "Check distgen generated files"
+    runs-on: ubuntu-20.04
+    if: |
+      github.event.issue.pull_request
+      && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: "refs/pull/${{ github.event.issue.number }}/head"
+          submodules: true
+
+      - name: Check distgen generated files
+        id: check
+        shell: bash
+        run: |
+          sha=$(git rev-parse HEAD)
+          sudo apt update && sudo apt -y install python3-pip
+          pip3 install pyyaml distgen Jinja2
```

Comment:
nitpick: pyyaml and Jinja2 are dependencies of distgen so they should be installed by default

Reply:
yes. This was problem in s2i-python-container. They, I do not know why, had not been installed automatically as dependencies and the job failed.

But I will remove them here and try to run the job.
If it fails, I will ad the dependencies afterwards.